### PR TITLE
Added a new haiku in bluewhale_cloud.md file

### PR DESCRIPTION
**Pull request by Gowtham-369**
It adds my custom japanese haiku stored in bluewhale_cloud.md file

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! It looks like the file path isn't quite right — can you ensure that it is in the `_haikus/` directory? Once you update the path we can merge this in.

*Please note: maintainers will only go through one round of review, first reviews that are not sufficiently addressed will be closed and marked invalid. Also note that after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by Gowtham-369** [comment]
Hi @ltagliaferri, I corrected the path by moving the .md file to _haikus folder as requested. Please approve my PR. Thanks :)
Link to changes commit: [https://github.com/do-community/cloud_haiku/pull/2944/commits/280c736530b3dae53f205482e9a4237385e1a77d](https://github.com/do-community/cloud_haiku/pull/2944/commits/280c736530b3dae53f205482e9a4237385e1a77d)

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**
